cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/StringFunctionsTest.scala

```diff
@@ -153,9 +154,10 @@ class StringFunctionsTest extends DocumentingTest {
     }
     section("toString()", "functions-tostring") {
       p(
-        """`toString()` converts an integer, float or boolean value to a string.""".stripMargin)
+        """`toString()` converts an integer, float, boolean, point, duration, date, time, localtime, localdatetime or datetime value to a string.""".stripMargin)
       function("toString(expression)", "A String.", ("expression", "An expression that returns a number, a boolean, or a string."))
```

Comment:
This line also needs to be updated with the temporal and spatial values. 

Also, the PRD says that `toString()` should have the signature `toString( value :: ANY? ) :: (STRING?)
` but that is not really true since e.g. toString of a list throws an error. Not sure if APOC has the same behaviour of their `toString()` or if it matters at this point, but I found it a bit odd.

Reply:
It has that signature. Doesn't mean that it can't throw for some of them

Reply:
Ok, just felt unsure when I checked the PRD and this PR about it